### PR TITLE
Filler post to set scene for Open AI

### PR DESCRIPTION
**Pull request by wpschaub**
Create a filler post to set the scene of our InnoFest idea to use Open AI to improve information awareness and doscoverability.

### CODE REVIEW

File: content/why bother with governance.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Title: Why bother with governance?
+Date: 2023-04-28
+Category: Posts 
+Tags: engineering, quality, learning
+Slug: why-bother-with-governance
+Author: Willy-Peter Schaub
+Summary: Why bother with governance when no-one can find it?
+
+ will not waste your time with the **why** and **what** of governance, guardrails, policies, and standards has been covered in [Servant stewardship of guardrails](/servant-stewardship-of-guardrails.html) and [Can we enact governance through engineering-friendly manifestos and guardrails](/governance-manifestos-guardrails.html). Instead, I want us to have a discussion as to why it is so difficult to encourage engineering to remain within guardrails, to collaborate to make improvements, and to steward the guardrails that are intended to keep us all from danger.
```
Comment (MerlinDarkstar) [inline content/why bother with governance.md]: I think you missed the leading "I" or "We"

Comment (MerlinDarkstar) [inline content/why bother with governance.md]: and standards has been covered
s/b
and standards that have been covered

Comment (MerlinDarkstar) [reply content/why bother with governance.md]: remain within guardrails, to collaborate to make 
... need additional comma
remain within guardrails, to collaborate, to make

File: images/why-bother-with-governance-2.png
Comment (MerlinDarkstar) [inline images/why-bother-with-governance-2.png:1]: LOVE this

File: content/why bother with governance.md (hunk 1 above)
Comment (wpschaub) [reply content/why bother with governance.md]: Fixed.

Comment (wpschaub) [reply content/why bother with governance.md]: Not sure what you mean. Double checked the sentence with Word.

**Review by MerlinDarkstar — DISMISSED** [review]

**Review by danielrbroderick — DISMISSED** [review]
Looks good

**Review by AndreKaminski — DISMISSED** [review]
Could you please obfuscate the QR code in the image. Otherwise good to go

**Comment by wpschaub** [comment]
> Could you please obfuscate the QR code in the image. Otherwise good to go

@AndreKaminski done in latest commit. Will publish tomorrow and update our biweekly report at the same time.

**Pull request merged**